docs/src/man/reshaping_and_pivoting.md

```diff
@@ -268,27 +303,27 @@ julia> d = stack(iris, view=true);
 
 julia> first(d, 6)
 6×4 DataFrame
-│ Row │ variable    │ value    │ Species       │ id    │
-│     │ Symbol      │ Float64⍰ │ Categorical…⍰ │ Int64 │
-├─────┼─────────────┼──────────┼───────────────┼───────┤
-│ 1   │ SepalLength │ 5.1      │ setosa        │ 1     │
-│ 2   │ SepalLength │ 4.9      │ setosa        │ 2     │
-│ 3   │ SepalLength │ 4.7      │ setosa        │ 3     │
-│ 4   │ SepalLength │ 4.6      │ setosa        │ 4     │
-│ 5   │ SepalLength │ 5.0      │ setosa        │ 5     │
-│ 6   │ SepalLength │ 5.4      │ setosa        │ 6     │
+│ Row │ variable    │ value │ Species     │ id    │
+│     │ Cat…        │ Any   │ String      │ Int64 │
```

Comment:
`Any` sounds like a bug in `stack`?

Reply:
Yes - there was a bud. I fix it in https://github.com/JuliaData/DataFrames.jl/pull/2145. We should merge this PR first, and then I will update #2145 to reflect the changes here.